framework/src/utils/PetscSupport.C

```diff
@@ -927,6 +933,22 @@ colorAdjacencyMatrix(PetscScalar * adjacency_matrix,
   ISColoringDestroy(&iscoloring);
 }
 
+void
+disableNonlinearConvergedReason(FEProblemBase & fe_problem)
+{
+  auto & petsc_options = fe_problem.getPetscOptions();
+
+  petsc_options.flags.erase("-snes_converged_reason");
+}
+
+void
+disableLinearConvergedReason(FEProblemBase & fe_problem)
+{
+  auto & petsc_options = fe_problem.getPetscOptions();
+
+  petsc_options.flags.erase("-ksp_converged_reason");
```

Comment:
Could you remind me what reason we can not print out the colorful converged reason?

Can we set up our own reason view?